shared/actions/platform-specific/index.native.tsx

```diff
@@ -423,7 +424,7 @@ const handleFilePickerError = (action: ConfigGen.FilePickerErrorPayload) => {
 const editAvatar = async () => {
   try {
     const result = await launchImageLibraryAsync('photo')
-    return result.cancelled === true
+    return result.cancelled
```

Comment:
this is different if cancelled is null/undefined

Reply:
this is typed as boolean. the new linting is TS aware